hathor/p2p/node_sync.py

```diff
@@ -355,23 +357,20 @@ def sync_from_timestamp(self, next_timestamp: int) -> Iterator[Deferred]:
 
         :param next_timestamp: Timestamp to start the sync
         """
-        self.log.debug('sync-{p} Sync starting at {next_timestamp}', p=self.short_peer_id,
-                       next_timestamp=next_timestamp)
+        self.log.debug('sync start', ts=next_timestamp)
```

Comment:
Why did you remove the peer id from all the logs in this file?

Reply:
I moved it to `self.log = logger.new(peer_id_short=self.short_peer_id)`, with that every call to `self.log.*` will include `peer_id_short`.

Reply:
Perfect, thanks!